config/default/common/config/wv.json/stories/severe-storm/atmospheric_rivers.json

```diff
@@ -0,0 +1,137 @@
+{
+  "stories": {
+    "atmospheric_rivers": {
+      "id": "atmospheric_rivers",
+      "type": "severe-storm",
+      "title": "Atmospheric Rivers",
+      "description": "Wintertime atmospheric rivers brought moisture to California.",
+      "backgroundImage": "atmo-rivers.png",
+      "backgroundImageHover": "",
+      "readMoreLinks": [
+        {
+          "title": "NASA Earth Observatory - A Boom Year for Sierra Nevada Snow ",
+          "link": "https://earthobservatory.nasa.gov/images/151232/a-boom-year-for-sierra-nevada-snow"
+        },
+        {
+          "title": "NASA Earth Observatory - Tulare Lake Grows",
+          "link": "https://earthobservatory.nasa.gov/images/151284/tulare-lake-grows"
+        },
+        {
+          "title": "NASA Earth Observatory - A Flood of Wildflowers",
+          "link": "https://earthobservatory.nasa.gov/images/151192/a-flood-of-wildflowers"
+        }
+      ],
+      "steps": [
+        {
+          "id": "001",
+          "description": "step001.html",
+          "transition": {
+            "element": "",
+            "action": ""
+          },
+          "stepLink": "v=-162.295938399713,16.42698668057487,-84.86031845217389,52.526006177450554&l=Reference_Features_15m,Coastlines_15m,VIIRS_SNPP_CorrectedReflectance_TrueColor&lg=true&t=2023-02-24-T10%3A00%3A00Z"
+        },
+        {
+          "id": "002",
+          "description": "step002.html",
+          "transition": {
+            "element": "",
+            "action": ""
+          },
+          "stepLink": "v=-162.295938399713,16.426986680574874,-84.86031845217389,52.526006177450554&l=Reference_Features_15m,Coastlines_15m,IMERG_Precipitation_Rate,VIIRS_SNPP_CorrectedReflectance_TrueColor&lg=true&t=2023-02-24-T10%3A00%3A00Z"
+        },
+        {
+          "id": "003",
+          "description": "step003.html",
+          "transition": {
+            "element": "animation",
+            "action": "play"
+          },
+          "stepLink": "v=-167.4294319852893,16.62622795596455,-75.64999743213335,59.92869114276513&z=4&ics=true&ici=5&icd=10&as=2023-02-24-T19%3A00%3A00Z&ae=2023-02-25-T03%3A00%3A00Z&l=Coastlines_15m,GOES-West_ABI_GeoColor,Land_Water_Map&lg=true&al=true&av=7.5&ab=on&t=2023-02-24-T19%3A00%3A00Z"
```

Comment:
For step 3, the map extent could be shifted down a bit - at least in my browser window, some of the atmospheric river was being cut off by the animation widget.

Reply:
I zoomed out a bit